scripts/monitor/notebook/plot-cromwell-bq-monitoring.ipynb

```diff
@@ -0,0 +1,1136 @@
+{
```

Comment:
<p>Bigquery -&gt; BigQuery.</p><p> </p><p>I find the explanation a bit convoluted, i.e. providing a bit too much detail. I'd suggest simply explain to the users</p><ul><li>this note books visualizes data from a BQ dataset that holds workflow resource usage monitoring data, and please refer to the Readme for what data are collected and how they are collected</li><li>you will get three plots (one PDF, two interactive HTML reports)</li><li>you need to provided us with </li><li class="ql-indent-1">workflow and sub-workflow IDs</li><li class="ql-indent-1">workflow execution dates</li><li class="ql-indent-1">where to store the output</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/broadinstitute/long-read-pipelines/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>done</p>